src/robothub_oak/application.py

```diff
@@ -159,6 +167,7 @@ def __manage_device(self) -> None:
                 else:
                     self.__manage_event.wait(25)
 
+            self.__watch_dog_device_reports.ping()
```

Comment:
The manage thread will sleep for 25 seconds in case of unsuccessful camera connection. Even if the camera is successfully connected, we can't be sure that 10 seconds is enough. We should probably change the logic here